cloud/aws-cloudwatch-metric-stream/aws_cloudwatch_metric_stream.tf

```diff
@@ -0,0 +1,59 @@
+data "aws_caller_identity" "current" {}
+data "aws_region" "current" {}
+
+resource "aws_s3_bucket" "sfx_metric_stream" {
+  bucket = "splunk-metric-streams-s3-${data.aws_caller_identity.current.account_id}-${data.aws_region.current.name}"
+}
+
+resource "aws_cloudwatch_log_group" "sfx_metric_stream" {
+  name              = "/aws/kinesisfirehose/splunk-metric-streams-${data.aws_region.current.name}"
+  retention_in_days = 14
```

Comment:
do you think it can be useful to provide a variable for configuration to the user ? the choice is up to you

Reply:
I checked and did not find any logs so far in both of the environments where I deployed this.

But why not, done.